libs/vkd3d/command.c

```diff
@@ -3482,9 +3482,9 @@ static void d3d12_command_list_debug_mark_end_region(struct d3d12_command_list *
         VK_CALL(vkCmdEndDebugUtilsLabelEXT(list->cmd.vk_command_buffer));
 }
 
-static void d3d12_command_list_clear_attachment_pass(struct d3d12_command_list *list, struct d3d12_resource *resource,
+static void d3d12_command_list_load_attachment(struct d3d12_command_list *list, struct d3d12_resource *resource,
         struct vkd3d_view *view, VkImageAspectFlags clear_aspects, const VkClearValue *clear_value, UINT rect_count,
-        const D3D12_RECT *rects, bool is_bound)
```

Comment:
Was is_bound always false? Seems like it from the diff at least.

Reply:
Yes, there was only ever one call to this with is_bound being false, so I removed it.